api/api_sources/schema-files/inspectApp.codes.schema.yaml

```diff
@@ -0,0 +1,83 @@
+version: '1.0'
+description: Schema file for various code tables associated with Inspect App dropdown fields
+externalTables: []
+includes: []
+schemas:
+  PreviousAISKnowledgeSourceSchema:
+    name: previous_ais_knowledge_source
+    description: 'Code table with options for sources of previous Aquatic Invasive Species (AIS) knowledge'
+    baseSchema: CodeTableSchema
+    meta:
+      baseModel: Record
+    displayLayout: {}
+    layout: {}
+    computedFields: {}
+    relations: {}
+    columns:
+      id:
+        name: previous_ais_knowledge_source_id
+        comment: Auto generated sequential primary key column.
+        definition: SERIAL PRIMARY KEY
+      description:
+        name: description
+        comment: 'Brief description of the knowledge source'
+        definition: VARCHAR(50) NOT NULL
+    versions: []
+    imports:
+      init:
+        fileName: PreviousAISKnowledgeSourceCode.csv
+        allColumns: true
+    fields: {}
+  ## -- end PreviousAISKnowledgeSourceSchema
+  PreviousInspectionSourceSchema:
+    name: previous_inspection_source
+    description: 'Code table with options for sources of previous invasive mussel inspections'
+    baseSchema: CodeTableSchema
+    meta:
+      baseModel: Record
+    displayLayout: {}
+    layout: {}
+    computedFields: {}
+    relations: {}
+    columns:
+      id:
+        name: previous_inspection_source_id
+        comment: Auto generated sequential primary key column.
+        definition: SERIAL PRIMARY KEY 
+      description:
+        name: description
+        comment: 'Brief description of the source of the previous inspection(s)'
+        definition: VARCHAR(50) NOT NULL
+    versions: []
+    imports:
+      init:
+        fileName: PreviousInspectionSourceCode.csv
+        allColumns: true
+    fields: {}
+  ## -- end PreviousInspectionSourceSchema
+  AdultMusselsFoundLocationSchema:
+    name: adult_mussels_found_location
```

Comment:
Recommended, don't use the verb in the model name. Example: adult_mussels_location (AdultMusselsLocation) | adult_mussels_location_info (AdultMusselsLocationInfo)